iree/turbine/kernel/wave/wave.py

```diff
@@ -352,8 +352,11 @@ def test_execute(self, args, kwargs):
                     kernel_outputs.append(arg)
 
             dynamic_symbols_map = kwargs.get("dynamic_symbols_map", {})
+            kernel_dynamic_dims = []
             if dynamic_symbols:
-                kernel_inputs += [dynamic_symbols_map[sym] for sym in dynamic_symbols]
+                kernel_dynamic_dims = [
```

Comment:
minor: Can we use dynamic_symbols_map.values() instead?

Reply:
nice idea, done!